BootloaderCorePkg/Library/MpInitLib/Ia32/MpInitLib.c

```diff
@@ -145,10 +184,10 @@ MpInit (
       AsmGetHotAddCodeAddressMap (&mAddressMap);
```

Comment:
Can we rename this function?

Reply:
OK.  I can rename it.  It is not part of my original code change.